### PR TITLE
Add org.sugarlabs.PyEyes

### PR DESCRIPTION
**Pull request by imhemish**
- [x] Please describe your application briefly.
PyEyes is a part of [sugar apps](https://sugarlabs.org). It is a simple activity where the eyes in the application follow the mouse cursor, much like xeyes.
- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I have [built][build] and tested the submission locally.
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub.

Migration guide enables anyone to port to flatpak.

- [X] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.

Yes the migration guide mentions the use of sugarlabs.org domain

- [ ] Any additional patches or files have been submitted to the upstream projects concerned.

The patches have not been upstreamed as SugarLabs flatpak migration guide states, that those are to come at later stage when sugarapps (the library enabling sugar apps to port to flathub) gets mature. (https://github.com/tchx84/sugarapp/blob/master/flatpak-guide.md)

### CODE REVIEW

File: org.sugarlabs.PyEyes.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            "name": "pyeyes",
+            "buildsystem": "simple",
+            "build-commands": [
+                "python3 setup.py install --prefix=${FLATPAK_DEST} --skip-install-desktop-file --skip-install-mime"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/sugarlabs/pyeyes-activity.git",
+                    "branch": "master"
```
Comment (hfiguiere) [inline org.sugarlabs.PyEyes.json:29]: git branch should NOT be used. You must use a commit / tag.

Comment (imhemish) [reply org.sugarlabs.PyEyes.json]: updated now

File: org.sugarlabs.PyEyes.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            "buildsystem": "simple",
+            "build-commands": [
+                "python3 setup.py install --prefix=${FLATPAK_DEST} --skip-install-desktop-file --skip-install-mime"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/sugarlabs/pyeyes-activity.git",
+		    "commit": "478fcaf51df5556a3fc26503dfbb63d9d13d1405"
+                },
```
Comment (bbhtt) [inline org.sugarlabs.PyEyes.json:30]: The indentation is wrong here
```suggestion
                    "url": "https://github.com/sugarlabs/pyeyes-activity.git",
		                "commit": "478fcaf51df5556a3fc26503dfbb63d9d13d1405"
                },
```

The indentation is wrong here

File: org.sugarlabs.PyEyes.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                    "path": "pyeyes-setup.patch"
+                }
+            ],
+            "post-install": [
+                "sugarapp-gen-mimetypes activity/activity.info mimetypes",
+                "sugarapp-gen-appdata activity/activity.info appdata",
+                "sugarapp-gen-desktop activity/activity.info desktop --mimetypes mimetypes",
+                "install -D mimetypes /app/share/mime/packages/org.sugarlabs.PyEyes.xml",
+                "install -D mimetypes /app/share/sugar/activities/PyEyes.activity/activity/mimetypes.xml",
+                "install -D appdata /app/share/metainfo/org.sugarlabs.PyEyes.appdata.xml",
```
Comment (bbhtt) [inline org.sugarlabs.PyEyes.json:54]:
```suggestion
                "install -D appdata /app/share/metainfo/org.sugarlabs.PyEyes.metainfo.xml",
```

https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#path-and-filename

**Comment by bbhtt** [comment]
cc @tchx84 as they are involved with sugar labs

**Comment by imhemish** [comment]
I just realised the update_contact should give the maintainer's email and not tch@sugarlabs.org as it belongs to @tchx84

**Comment by imhemish** [comment]
I will correct all these shortly

**Comment by tchx84** [comment]
@imhemish let's coordinate on Sugar Labs's matrix.

**Comment by hfiguiere** [comment]
also the base app uses a runtime that will be EOL real soon now....

**Comment by tchx84** [comment]
@imhemish following up the conversation we had on matrix, I suggest we close this PR for now.

**Comment by imhemish** [comment]
> @imhemish following up the conversation we had on matrix, I suggest we close this PR for now.

Done

**Pull request closed without merging**